pom.xml

```diff
@@ -217,7 +168,6 @@
                     <plugin>
                         <groupId>org.codehaus.mojo</groupId>
                         <artifactId>findbugs-maven-plugin</artifactId>
-                        <version>3.0.2</version>
                         <configuration>
                             <xmlOutput>true</xmlOutput>
                             <failOnError>false</failOnError>
```

Comment:
Could probably be deleted.

BTW with the `findbugs` profile (see a few lines up), I am setting up the POM so that when you

    mvn -DskipTests clean install

(i.e., a normal developer build) you do not pay the price of running FindBugs analysis, which can be quite slow: you do the minimum necessary to get artifacts, and perhaps later run _selected_ tests.

CI builds, or careful developer builds, do not set `skipTests` and so we both run tests _and_ static analysis. Perhaps an abuse of `skipTests` but quite convenient.

Also the `maven-javadoc-plugin` execution here is perhaps superseded by your inclusion of this in the release profile, though I recommend the parent POM be made to do it the way I do here: so that CI builds, or local builds in “careful” mode, verify the absence of Javadoc errors (especially on JDK 8). This ensures that mistakes are caught during PRs, well before an attempt is made to cut a release.